BrotherPrinter/app.json

```diff
@@ -0,0 +1,10 @@
+{
+    "appid": "4aad630e6c72b64c79df498803ba2247372bbba7",
+    "name": "Brother Printer",
+    "website": "https://www.brother-usa.com/home/printers-fax",
+    "license": "GNU General Public License v3.0 only",
```

Comment:
I highly doubt that the Brother web interface is GPL v3. Couldn't find it myself at least.

Reply:
Yes, just a link to the printer-UI.